notebooks/cPython_wrapper.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
I think this is an excellent utility to include in the project for future validation.  A description of the process times will help to isolate the long running processes and help identify timing bottlenecks. 

One note for future ideas on this, it may be worth looking into creating "Utility" or "Helper" folder for functions along these lines that help with our processes but are not app centric, rather than keeping them with the notebooks or other parts of the codebase.

Reply:
There's also the time function in python, but I found this to be better at isolating specific calls so we could see both the amount and how long each takes.